Unix/tools/auth_tests_setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+#
+# Used by regress and 'make tests'.  Should be run at omi/Unix level.
```

Comment:
Worth doing a basename on the current dir and validating that we are truly at the `omi\Unix` level?